shared/src/commonMain/kotlin/App.kt

```diff
@@ -4,17 +4,21 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.graphics.ImageBitmap
 import androidx.lifecycle.Lifecycle
 import androidx.lifecycle.compose.LifecycleEventEffect
+import at.asitplus.wallet.app.common.DCAPIInvocationData
 import at.asitplus.wallet.app.common.WalletMain
 import io.github.aakira.napier.Napier
 import kotlinx.coroutines.flow.MutableStateFlow
 import ui.navigation.WalletNavigation
 import ui.theme.WalletTheme
+import ui.viewmodels.authentication.PresentationStateModel
 
 /**
- * Global variable which especially helps to channel information from swift code
+ * Global variables which especially help to channel information from swift code
```

Comment:
That comment is not correct, and only shown for `appLink`. Please also add for `dcapiInvocationData` and `presentationStateModel`